### ISSUE TITLE
Guided Tours - mobile

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40011]
### Steps to reproduce the issue
See video recording - keyboard navigation only - all key presses highlighted on screen by external software
Creating seperate issue for each bug report so that they dont get lost inside a bigger report (again)

No tours work on a small screen as the New button is obscured by the step information

https://user-images.githubusercontent.com/1296369/222924701-54bfe206-da8b-44f3-bd3d-cfec93b92cd4.mp4

The requirement to use a mouse to successfully use the Guided Tours component is an accessibility failure and breaks our commitment to providing an accessible way to use Joomla

**Comment by wojtekxtx** [comment]
@brianteeman These modals should be moveable.

**Comment by brianteeman** [comment]
@wojtekxtx please try to respond to the actual reported issue.

**Comment by Quy** [comment]
OK to close per #40045?

**Comment by wojtekxtx** [comment]
@Quy Think so 👍
